docs/testplan/build_2tier_network.md

```diff
@@ -0,0 +1,386 @@
+
+### Configuration
+
+For example, assume T0-A's Ethernet256 is connected to T1-A's Ethernet0. To establish a BGP session between a T0 switch (T0-A) and a T1 switch (T1-A), the following configurations must be applied.
+
+#### Enable BGP Feature
+
+First, BGP must be enabled on both T0 and T1 switches to allow them to participate in BGP-based route exchange.
+
+T0-A Configuration:
+
+```json
+"FEATURE": {
+    "bgp": {
+        "auto_restart": "enabled",
+        "check_up_status": "false",
+        "delayed": "False",
+        "has_global_scope": "False",
+        "has_per_asic_scope": "True",
+        "high_mem_alert": "disabled",
+        "set_owner": "local",
+        "state": "enabled",
+        "support_syslog_rate_limit": "true"
+    }
+}
+```
+
+T1-A Configuration:
+
+```json
+"FEATURE": {
+    "bgp": {
+        "auto_restart": "enabled",
+        "check_up_status": "false",
+        "delayed": "False",
+        "has_global_scope": "False",
+        "has_per_asic_scope": "True",
+        "high_mem_alert": "disabled",
+        "set_owner": "local",
+        "state": "enabled",
+        "support_syslog_rate_limit": "true"
+    }
+}
+```
+
+#### Define Device Metadata
+
+Define the metadata of the two devices. Each switch must be correctly identified with its BGP Autonomous System Number (ASN) and router ID. T0 switches function as top-of-rack routers, while T1 switches serve as leaf routers.
+
+T0-A Configuration:
+
+```json
+"DEVICE_METADATA": {
+    "localhost": {
+        "bgp_asn": "64001",
+        "buffer_model": "traditional",
+        "cloudtype": "Public",
+        "default_bgp_status": "up",
+        "default_pfcwd_status": "enable",
+        "deployment_id": "1",
+        "docker_routing_config_mode": "separated",
+        "hostname": "switch-T0-A",
+        "hwsku": "ABCDEFG",
+        "region": "None",
+        "synchronous_mode": "enable",
+        "timezone": "UTC",
+        "type": "ToRRouter",
+        "yang_config_validation": "disable",
+        "bgp_router_id": "10.100.0.81"
+    }
+}
+```
+
+T1-A Configuration:
+
+```json
+"DEVICE_METADATA": {
+    "localhost": {
+        "bgp_asn": "65001",
+        "buffer_model": "traditional",
+        "cloudtype": "Public",
+        "default_bgp_status": "up",
+        "default_pfcwd_status": "enable",
+        "deployment_id": "1",
+        "docker_routing_config_mode": "separated",
+        "hostname": "switch-T1-A",
+        "hwsku": "ABCDEFG",
+        "region": "None",
+        "synchronous_mode": "enable",
+        "timezone": "UTC",
+        "type": "LeafRouter",
+        "yang_config_validation": "disable",
+        "bgp_router_id": "10.100.0.88"
+    }
+}
+```
+
+#### Configure Ports and Interfaces
+
+Each BGP session requires a pair of interfaces. Configure the two interfaces, ensuring they are in the same subnet.
+
+T0-A Configuration:
```

Comment:
same here, will be better to keep the setup short - only shows what needs to be done.

e.g. in this case, we need to assign the interface ip to the port, which should be part of the topology setting. Using a graph I mentioned above and show the essence will be good enough.